src/nvim/syntax.c

```diff
@@ -7476,6 +7476,34 @@ int highlight_changed(void)
   return OK;
 }
 
+bool parse_winhl(win_T *wp)
```

Comment:
Should this go in option.c? It is parsing options, and it's only used there as well.